Automattic/wp-calypso
# Jetpack blocks: Remove Gutenlypso code

#### Changes proposed in this Pull Request

Remove Gutenlypso-specific functionality from Jetpack blocks

#### Testing instructions

* Does Calypso continue to build and run well?
* Do the blocks work well?
* Does the [`build-jetpack-blocks` job](https://circleci.com/gh/Automattic/wp-calypso/215593#artifacts/containers/0) appear to have the right artifacts?
